### PR TITLE
Configured pom.xml, application.properties, created Book model, Book Service and Book Repository 

### PR DESCRIPTION
**Pull request by OlegTuryk**

### CODE REVIEW

File: src/main/java/com/app/onlinebookstore/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    private String title;
+    @Column(nullable = false)
+    private String author;
+    @Column(nullable = false, unique = true)
+    private String isbn;
+    @Column(nullable = false)
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
+
```
Comment (irynamekh) [inline src/main/java/com/app/onlinebookstore/model/Book.java]: an extra empty line here

File: src/main/java/com/app/onlinebookstore/repository/impl/BookRepositoryImpl.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            session = sessionFactory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(book);
+            transaction.commit();
+            return book;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't insert book " + book, e);
```
Comment (irynamekh) [inline src/main/java/com/app/onlinebookstore/repository/impl/BookRepositoryImpl.java]: it is better to create your own error, I have seen this correction from mentors among colleagues

**Review by irynamekh — CHANGES_REQUESTED** [review]

**Review by irynamekh — APPROVED** [review]

File: src/main/java/com/app/onlinebookstore/model/Book.java (hunk 1 above)
Comment (vovasalyha) [inline src/main/java/com/app/onlinebookstore/model/Book.java]:
```suggestion
```

File: src/main/java/com/app/onlinebookstore/repository/impl/BookRepositoryImpl.java (hunk 2 above)
Comment (vovasalyha) [inline src/main/java/com/app/onlinebookstore/repository/impl/BookRepositoryImpl.java]: Use more purposeful exception (or create your own)

**Review by vovasalyha — APPROVED** [review]
GJ! Please take a look on minor comments

**Pull request merged**